src/components/Stats/Stats.js

```diff
@@ -64,7 +64,7 @@ const Stats = () => {
           type="primary"
           shape="round"
         >
-          <Link to="/incidents">Incidents</Link>
+          <Link to="/incident-reports">Reports</Link>
```

Comment:
I think we should make the link itself say "incident-reports" not just "reports"